hushline/routes.py

```diff
@@ -87,13 +87,15 @@ def inbox() -> Response | str:
         )
         secondary_users_dict = {su.id: su for su in primary_user.secondary_usernames}
 
-        return render_template(
-            "inbox.html",
-            user=primary_user,
-            secondary_username=None,
-            messages=messages,
-            is_secondary=False,
-            secondary_usernames=secondary_users_dict,
+        return (
+            render_template(
+                "inbox.html",
+                user=primary_user,
+                secondary_username=None,
+                messages=messages,
+                is_secondary=False,
+                secondary_usernames=secondary_users_dict,
+            ),
```

Comment:
This is returning a 1-tuple of of type `(str, )`. Drop the outer parenthesis and comma.

Reply:
👍